libs/StatusGoQt/src/StatusGo/Accounts/AccountsAPI.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
How does it differ from `Accounts.h` in its purpose?

Reply:
Accounts are the previous implementation and need refactoring to align with the typed wallet approach. After the refactoring of `Account.h`, its content will be moved here, and this file might get split in two if too big.